src/oclif/hooks/init.ts

```diff
@@ -1,50 +1,53 @@
-#!/usr/bin/env node
-const initTimeStartTime = process.hrtime()
-
-import 'v8-compile-cache'
-
 import axios from 'axios'
 import chalk from 'chalk'
-import { all as clearCachedModules } from 'clear-module'
-import { CommandNotFoundError, find, MissingRequiredArgsError, run as unboundRun } from 'findhelp'
 import os from 'os'
-import path from 'path'
-import { without } from 'ramda'
-import * as pkg from '../package.json'
-import { CLIPreTasks } from './CLIPreTasks/CLIPreTasks'
-import * as conf from './conf'
-import { envCookies } from './env'
-import { CommandError, SSEConnectionError, UserCancelledError } from './errors'
-import { Token } from './lib/auth/Token'
-import { TelemetryCollector } from './lib/telemetry/TelemetryCollector'
-import log from './logger'
-import tree from './modules/tree'
-import { checkAndOpenNPSLink } from './nps'
-import notify from './update'
-import { isVerbose, VERBOSE } from './utils'
-import { Metric } from './lib/metrics/MetricReport'
-import { hrTimeToMs } from './lib/utils/hrTimeToMs'
-
-const run = command => Promise.resolve(unboundRun.call(tree, command, path.join(__dirname, 'modules')))
+import { all as clearCachedModules } from 'clear-module'
```

Comment:
I think maybe this package is not necessary anymore, it could even be removed from `package.json`

Reply:
Could you explain why? I didn't get it.

Reply:
`clear-module` - https://www.npmjs.com/package/clear-module -  is used to reset a imported module, so everything inside it will be reexecuted. This is used because some things are executed in the root of the module, which I think it's not such a good practice (these things are executed on the first `import` of the module - every other import is cached - clear-modules will clean this cache so it will be executed again)

https://github.com/vtex/toolbelt/blob/6240a1932c0c7e11f00054a12b20a5fc692b2db6/src/modules/auth/login.ts#L11-L13

Analyzing the things executed in the root of this module I think it would be ok if we didn't cleared the cache before calling the login function again (In the future it would be ideal to move the logic in the root of the module to the login function)

Also, it's always a good thing to remove dependencies = reduced import times (every import costs us init time) and slighly less npm package size

Reply:
Nice!!